### ISSUE TITLE
RestyResolver incorrect routing with in-path variables

### ISSUE THREAD

**Issue by amyasnikov** [spec-first/connexion#1165]
### Description
If we add the following to the standard Pet Store example:
```
/pets/{petId}/owner:
    get:
      parameters:
        - name: petId
          in: path
          required: true
          schema:
            type: string
      responses:
        '200':
          content:
            application/json:
              schema:
                type: object
```

then we get the following error after running the app:
```
Traceback (most recent call last):
  File "./resty.py", line 13, in <module>
    resolver=RestyResolver('api'))
  File "/usr/local/lib/python3.7/dist-packages/connexion/apps/flask_app.py", line 58, in add_api
    self.app.register_blueprint(api.blueprint)
  File "/usr/local/lib/python3.7/dist-packages/flask/app.py", line 98, in wrapper_func
    return f(self, *args, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/flask/app.py", line 1167, in register_blueprint
    blueprint.register(self, options, first_registration)
  File "/usr/local/lib/python3.7/dist-packages/flask/blueprints.py", line 256, in register
    deferred(state)
  File "/usr/local/lib/python3.7/dist-packages/flask/blueprints.py", line 294, in <lambda>
    self.record(lambda s: s.add_url_rule(rule, endpoint, view_func, **options))
  File "/usr/local/lib/python3.7/dist-packages/flask/blueprints.py", line 86, in add_url_rule
    **options
  File "/usr/local/lib/python3.7/dist-packages/flask/app.py", line 98, in wrapper_func
    return f(self, *args, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/flask/app.py", line 1283, in add_url_rule
    "existing endpoint function: %s" % endpoint
AssertionError: View function mapping is overwriting an existing endpoint function: /v1_0.api_pets_get
```

This happens always if you have something to route after in-path parameter (petId).

### Expected behaviour

`/pets/{petId}/owner get` has been routed to operationId **api.pets.owner.get**

### Actual behaviour

`/pets/{petId}/owner get` has been routed to operationId **api.pets.get**

### Steps to reproduce

Add  `/pets/{petId}/owner` to restyresolver example

### Additional info:

- Python 3.7.3
- pip3 show connexion | grep "^Version\:"
Version: 2.6.0

**Comment by lpk1950** [comment]
Facing the same issue. Any fixes already ?

**Comment by RobbeSneyders** [comment]
Which version are you using @lpk1950? This should be fixed since 2.8.0.
